tests/core/pyspec/eth2spec/test/helpers/rewards.py

```diff
@@ -219,7 +221,8 @@ def run_get_inactivity_penalty_deltas(spec, state):
 
 def transition_state_to_leak(spec, state, epochs=None):
     if epochs is None:
-        epochs = spec.MIN_EPOCHS_TO_INACTIVITY_PENALTY
+        # +1 to trigger leak_score transitions
+        epochs = spec.MIN_EPOCHS_TO_INACTIVITY_PENALTY + 1
```

Comment:
Sanity check, what exactly the `@leaking` decorator should do? Is it for getting a `state` such that `is_in_inactivity_leak(state) is True`?

If so, should this line be `epochs = spec.MIN_EPOCHS_TO_INACTIVITY_PENALTY + 2`, similar to `test_leak_scores` test case in `lightclient_patch/sanity/test_blocks.py`?

Reply:
So @leaking used to just get the state such taht `is_in_inactivity_leak` is True, but this is not enough for the test cases that use `leak_scores` because on the cusp of the transition, `is_in_inactivity_leak is False` and thus `leak_scores` aren't updated to be non-zero. Whereas once you cross the next transition, you do get increased leak scores. So beacuse we want to test that running the rewards/penalties functions in isolation gives leak penalties, we want some leak_scores to be non-zero

the +2 in `test_leak_scores` is to account for if it is starting at genesis where `finalized_epoch == current_epoch` so it takes another couple of epoch transitions to get to the leak